resources/lang/bn.json

```diff
@@ -35,13 +35,9 @@
     "Create Role": "রোল তৈরি করুন",
     "Create User": "ব্যবহারকারী তৈরি করুন",
     "CRM": "সিআরএম",
-    "CRM Dashboard": "সিআরএম ড্যাশবোর্ড",
-    "Customers": "গ্রাহক",
-    "New Customer": "নতুন গ্রাহক",
     "Custom CSS & JavaScript": "কাস্টম সিএসএস এবং জাভাস্ক্রিপ্ট",
     "Dark Mode": "ডার্ক মোড",
     "Dark Mode Colors": "ডার্ক মোড রঙ সমূহ",
-    "Dashboard": "ড্যাশবোর্ড",
```

Comment:
Can you check manually, why you're removing existing translations?

check for others too.
Cause Dashboard is for core too.